test/smoke/src/sql/areas/notebook/notebookView.test.ts

```diff
@@ -9,21 +9,33 @@ import * as minimist from 'minimist';
 import { afterSuite, beforeSuite } from '../../../utils';
 
 export function setup(opts: minimist.ParsedArgs) {
-	describe('NotebookView', () => {
+	describe('NotebookTreeView', () => {
 		beforeSuite(opts);
 		afterSuite(opts);
 
-		it('Pin a notebook', async function () {
+		it('Pin notebook', async function () {
 			const app = this.app as Application;
 			await app.workbench.sqlNotebook.view.focusNotebooksView();
-			await app.workbench.sqlNotebook.view.pinNotebook();
-			await app.workbench.sqlNotebook.view.waitForPinnedNotebookView();
+			const sqlNotebook = (await app.workbench.sqlNotebook.view.getNotebookTreeItems()).filter(n => n.textContent === 'collapsed');
```

Comment:
Make `collapsed` a const and clearly indicate that it's the name of a Notebook in the smoke test repo please!